src/components/DataCollector/index.js

```diff
@@ -53,13 +53,9 @@ const createDataCollector = ({ eventManager, logger }) => {
           event.setUserData(data);
 
           const details = {
-            isViewStart: viewStart
+            renderDecisionsEnabled: renderDecisions,
```

Comment:
If we prefer the `Enabled` suffix for ourselves, should we prefer the `Enabled` suffix on the public API on behalf of our customers? Should it be `decisionRenderingEnabled` instead of `renderDecisionsEnabled`? Looking at some of the other configuration options we have:

* clickCollectionEnabled
* cookieDestinationsEnabled
* debugEnabled

I think `decisionRenderingEnabled` might be the most appropriate, though one could make the argument that `debugEnabled` should be `debuggingEnabled`. I hate naming things.

Reply:
I think that options passed in the event command should be named `renderDecisions`,`viewStart` etc., anything related to configuration settings should be suffixed with `enabled` and we should follow the convention like `debuggingEnabled`, `cookieDestinationEnabled` etc.

Reply:
👍  I do like `renderDecisions` more than `decisionRenderingEnabled`. Maybe I like it more because "enabled" sounds like a long-running configuration option rather than whether something should be done a single time (in this case, decisions should be rendered). This would seem to agree with what you've stated.

I think I'm okay with the configuration options as they are though.

Reply:
ok, then I'll rename it to `renderDecisions`